roles/ceilometer-common/defaults/main.yml

```diff
@@ -38,26 +38,48 @@ ceilometer:
   heartbeat_timeout_threshold: 30
   collector_workers: 4
   logs:
-    - paths:
-        - /var/log/ceilometer/api.log
-      fields:
-        type: openstack
-        tags: ceilometer,ceilometer-api
-    - paths:
-        - /var/log/ceilometer/agent-notification.log
-      fields:
-        type: openstack
-        tags: ceilometer,ceilometer-agent-notification
-    - paths:
-        - /var/log/ceilometer/collector.log
-      fields:
-        type: openstack
-        tags: ceilometer,ceilometer-collector
-    - paths:
-        - /var/log/ceilometer/polling.log
-      fields:
-        type: openstack
-        tags: ceilometer,ceilometer-polling
+    ubuntu:
```

Comment:
/usr/lib/systemd/system/openstack-ceilometer-api.service, package default installation define the logfilename.  but our rhosp version remove that parameter.  I just add the filename with *api.log to match both api.log and ceilometer-api.log